tests/dragonfly/sentinel_test.py

```diff
@@ -43,32 +44,29 @@ async def await_for(func, pred, timeout_sec, timeout_msg=""):
 
 
 class Sentinel:
-    def __init__(self) -> None:
+    def __init__(self, config_dir) -> None:
+        self.config_file = pathlib.Path(config_dir).joinpath("sentinel.conf")
         self.port = 5555
         self.image = "bitnami/redis-sentinel:latest"
         self.container_name = "sentinel_test_py_sentinel"
         self.default_deployment = "my_deployment"
         self.initial_master_port = 1111
+        self.proc = None
 
     def start(self):
-        self.stop() # cleanup potential zombies
-
-        cmd = [
-            "docker", "run",
-            "--network=host",
-            "-d",
-            f"--name={self.container_name}",
-            "-e", "REDIS_MASTER_HOST=localhost",
-            "-e", f"REDIS_MASTER_PORT_NUMBER={self.initial_master_port}",
-            "-e", f"REDIS_MASTER_SET={self.default_deployment}",
-            "-e", f"REDIS_SENTINEL_PORT_NUMBER={self.port}",
-            "-e", "REDIS_SENTINEL_QUORUM=1",
-            "-e", "REDIS_SENTINEL_DOWN_AFTER_MILLISECONDS=3000",
-            "bitnami/redis-sentinel:latest"]
-        assert subprocess.run(cmd).returncode == 0, "docker run failed."
+        config = [
+            f"port {self.port}",
+            f"sentinel monitor {self.default_deployment} 127.0.01 {self.initial_master_port} 1",
```

Comment:
`127.0.01`  is a typo

Reply:
@romange thanks, fixed (somehow sentinel picked up the master on localhost and test was passing regardless)